packages/electron/src/main/builder/BacktraceClientBuilder.ts

```diff
@@ -0,0 +1,10 @@
+import { BacktraceNodeClientSetup, BacktraceClientBuilder as NodeBacktraceClientBuilder } from '@backtrace-labs/node';
+import { BacktraceClient } from '../BacktraceClient';
+
+export class BacktraceClientBuilder extends NodeBacktraceClientBuilder {
```

Comment:
can we provide more attribute providers? 

We can get at least:
- application name
- application version
- OS culture
- app metrics 
- GPU information
- check if electron is running in the unity env
- is application packaged?
- window information like fullscreen, window size, kiosk mode etc
- information if the app is online


Definitely the app might not behave correctly if we package it because of lack of application name/version that we should expose via electron - node can fail here

Reply:
Added additional providers.